contracts/token/ERC1155/Tatum1155.sol

```diff
@@ -236,20 +218,12 @@ contract Tatum1155 is Context, ERC165, IERC1155, IERC1155MetadataURI {
 
             uint256 fromBalance = _balances[id][from];
             require(
-                fromBalance > amounts[i],
+                fromBalance >= amounts[i],
```

Comment:
was this working finally?

Reply:
Yes , the cashbacks exhausted the balance if the msg.value was low in the loop but the error reported was this one instead, it is working now